docs/questions/native-editor/sql-parameters.md

```diff
@@ -141,6 +141,17 @@ A MongoDB native query example might look like this:
 
 For a more in-depth guide, check out [Field Filters: create smart filter widgets for SQL questions][field-filter].
 
+### Field filters in BigQuery and Oracle
+
+Make sure your SQL dialect matches the database you've selected. Common issues:
+
+| Database | Do this                    | Avoid                |
+| -------- | -------------------------- | -------------------- |
+| BigQuery | `` FROM `dataset.table` `` | `FROM dataset.table` |
+| Oracle   | `FROM "schema"."table"`    | `FROM schema.table`  |
```

Comment:
Original is fine, but I wonder if we can be more helpful than "spot the difference between do/avoid examples" by explaining what the quirk is, and giving an example that is correct (and not muddying things with incorrect examples).

This would make the table faster for people to read and realize, oh, I just to backtick it.

Something like: 

```suggestion
Make sure your SQL dialect matches the database you've selected. Common issues involving how tables are quoted in the query:

| Database | Dialect quirk              | Example               |
| -------- | -------------------------- | -------------------- |
| BigQuery | Schemas and tables must be quoted with backticks. | `` FROM `dataset.table` `` |
| Oracle   | Schemas and tables must be quoted in double quotes.   | `FROM schema.table`  |
```

Good call on the heading, by the way.